### PR TITLE
Pin Drone to Specific Image Version

### PR DESCRIPTION
**Pull request by Hamms**
Currently, we always use the latest uploaded docker image for drone.  This makes it difficult to test changes, since images need to be uploaded to be tested against drone. Rather than continue to test on personal dockerhub accounts, I suggest we simply pin to a version in the drone config.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Comment by cat5inthecradle** [comment]
This google doc should probably be updated to describe how we might go about updating the image.

https://docs.google.com/document/d/1Qls20xfNN6T_DErOMwVQOFJsxAEAdMWFHzBDIbxyUQQ/edit?disco=AAAAXU3wWrA

**Comment by Hamms** [comment]
That process is documented here: https://github.com/code-dot-org/code-dot-org/tree/staging/docker#updating-the-docker-hub-image and I don't believe has changed

**Comment by Hamms** [comment]
It does seem like the Google Doc and Git Readme do each give only half the picture, and neither is particularly discoverable from the other. There's definitely some room for improvement in this documentation space, but I'm going to say that addressing that is out of scope for this PR :smile:

File: .drone.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@ steps:
       filename: unit-tests-cache.tar
 
   - name: unit-tests
-    image: codedotorg/code-dot-org:latest
+    image: codedotorg/code-dot-org:1.2
```
Comment (sureshc) [inline .drone.yml:29]: We also need to update the Image version specified for the User Interface container (line 123).

**Review by sureshc — APPROVED** [review]

**Pull request merged**